File: pigatize/pigatize.rb
def pigatize(text)
  text = text.downcase
  @vowels = ["a", "e", "i", "o", "u"]
  if starts_with_vowel?(text)
    text = text + "way"
  else
    text = text[1..-1] + "#{text[0]}ay"
  end
end

def starts_with_vowel?(text)
  if @vowels.include?(text[0])
    true
  end
end

loop do
  puts "Please enter a word and I will translate to pig latin"
  text = gets.chomp
  break if text.length == 0 # Break out of the loop if I say nothing
  puts pigatize(text)
end

File: pigatize/pigatize.js
var vowels = ["a", "e", "i", "o", "u"]

var pigLatin = function(text) {
  for (var i = 0; i <= vowels.length; i++)
    if (text.charAt(0) === vowels[i])
      return text + 'way';
    else
      return text.substr(1) + text.substr(0,1) + 'ay';
}

console.log(pigLatin('apple'));
console.log(pigLatin('test'));





